jdbc/src/main/java/org/apache/zeppelin/jdbc/JDBCInterpreter.java

```diff
@@ -537,16 +555,30 @@ private boolean isDDLCommand(int updatedCount, int columnCount) throws SQLExcept
         }
       }
 
+      if (!quoteString && !doubleQuoteString && !multiLineComment && !singleLineComment
+          && sql.length() > item + 1) {
+        if (character.equals('-') && sql.charAt(item + 1) == '-') {
+          singleLineComment = true;
+          continue;
+        }
+
+        if (character.equals('/') && sql.charAt(item + 1) == '*') {
+          multiLineComment = true;
+          continue;
+        }
```

Comment:
single because `singleLineComment == true` will not yet a symbol of the new string

` if (singleLineComment || multiLineComment) {
        continue;
      }`